### PR TITLE
345 - improve PDF Terms layout

### PR DESCRIPTION
**Pull request by caravinci**
- Fixes the PDF styling to match the changes in the Terms section.
- Adds a new link, as per the ticket.
- Adds a missing period at the end of a paragraph.

### CODE REVIEW

File: components/pages/Applications/PDF/StaticTerms.tsx [hunk 1]
```diff
@@ -17,6 +19,7 @@ const StaticTerms = ({ isPdf = false, data }: { isPdf?: boolean; data?: Applicat
     ContainerComponent,
     LinkComponent,
     ListComponent,
+    UnorderedListComponent,
```
Comment (caravinci) [inline components/pages/Applications/PDF/StaticTerms.tsx:22]:
```suggestion
```

File: components/pages/Applications/PDF/StaticTerms.tsx [hunk 2]
```diff
@@ -82,37 +87,42 @@ const StaticTerms = ({ isPdf = false, data }: { isPdf?: boolean; data?: Applicat
...
-          project will be checked for conformity with the goals and policies of ICGC including, but
-          not limited to, policies concerning the purpose and relevance of the research, the
-          protection of the donors and the security of the donors’ data. If your application is
-          approved, you agree that your applicant's name, institution, and scientific lay summary
-          may be included in a registry of all projects that have been granted access to ICGC
-          Controlled Data. The ICGC DACO approved projects are posted on the{' '}
+          project will be checked for conformity with the{' '}
+          <LinkComponent href={POLICIES_PAGE} rel="noopener noreferrer" target="_blank">
+            goals and policies of ICGC
+          </LinkComponent>{' '}
```
Comment (caravinci) [inline components/pages/Applications/PDF/StaticTerms.tsx:112]: new link

File: components/pages/Applications/PDF/StaticTerms.tsx [hunk 3]
```diff
@@ -82,37 +87,42 @@ const StaticTerms = ({ isPdf = false, data }: { isPdf?: boolean; data?: Applicat
...
+          including, but not limited to, policies concerning the purpose and relevance of the
+          research, the protection of the donors and the security of the donors’ data. If your
+          application is approved, you agree that your applicant's name, institution, and scientific
+          lay summary may be included in a registry of all projects that have been granted access to
+          ICGC Controlled Data. The ICGC DACO approved projects are posted on the{' '}
           <LinkComponent
-            href="https://www.icgc-argo.org/page/139/controlled-data-users"
+            href={CONTROLLED_DATA_USERS_PAGE}
```
Comment (caravinci) [inline components/pages/Applications/PDF/StaticTerms.tsx:119]: reuse constants for this existing link

File: components/pages/Applications/PDF/StaticTerms.tsx [hunk 4]
```diff
@@ -82,37 +87,42 @@ const StaticTerms = ({ isPdf = false, data }: { isPdf?: boolean; data?: Applicat
...
             rel="noopener noreferrer"
             target="_blank"
           >
             ICGC ARGO website
           </LinkComponent>
+          .
```
Comment (caravinci) [inline components/pages/Applications/PDF/StaticTerms.tsx:125]: missing period

Comment (anncatton) [reply components/pages/Applications/PDF/StaticTerms.tsx:125]: I think this looks like the `.` is contained within a `TextComponent`, but just verifying! If it's not, the pdf will error.

Comment (caravinci) [reply components/pages/Applications/PDF/StaticTerms.tsx:125]: Good point! and that's right: I've checked that producing a PDF still works on my local.

**Review by anncatton — APPROVED** [review]

**Pull request merged**